pkg/operator/controller/ingress/deployment.go

```diff
@@ -777,7 +784,7 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 			}
 			if accessLogging.Destination.Syslog.MaxLength > 0 {
 				env = append(env, corev1.EnvVar{
-					Name:  RouterSyslogMaxLengthEnvName,
+					Name:  RouterLogMaxLengthEnvName,
```

Comment:
If we only have one environment variable for log length, then we need to have either the container or the syslog set it, not both.  E.g. if someone had syslog.maxlength set to 256, and then decided to use container logging, and set that to 300, then the change would be overwritten here.

I think you need to do something like:

```suggestion
if accessLogging.Destination.Container.MaxLength > 0 {
		env = append(env, corev1.EnvVar{
				Name:  RouterLogMaxLengthEnvName,
				Value: fmt.Sprintf("%d", accessLogging.Destination.Container.MaxLength),
		})
} else if if accessLogging.Destination.Syslog.MaxLength > 0 {
....
```

Reply:
My mistake - I didn't see the `case` statement.  Please disregard.   If you make other changes, can you add a blank line before the `case` statement here?